model/sbvr/supervisor.sbvr

```diff
@@ -0,0 +1,298 @@
+
+	Fact type: device has name
+		Necessity: each device has exactly one name.
+
+	Fact type: device has uuid
+		Necessity: each device has exactly one uuid.
+		Necessity: each uuid is of exactly one device.
+		Necessity: each device has a uuid that has a Length (Type) that is equal to 32.
+
+	Fact type: device has hostname
+		Necessity: each device has exactly one hostname.
+
+	Fact type: device is connected to vpn
+
+
+
+-- DEVICE METRICS
+
+Term: cpu id
+	Concept Type: Short Text (Type)
+
+Term: cpu temp
+	Concept Type: Integer (Type)
+
+Term: cpu usage
+	Concept Type: Integer (Type)
+
+Term: memory total
+	Concept Type: Integer (Type)
+
+Term: memory usage
+	Concept Type: Integer (Type)
+
+Term: storage block device
+	Concept Type: Short Text (Type)
+
+Term: storage total
+	Concept Type: Integer (Type)
+
+Term: storage usage
+	Concept Type: Integer (Type)
+
+Term: metric
+-- Separate metrics from other device info for greater modularity
+
+	Fact type: device provides metric
+	-- Is this necessary?
+		Synonymous Form: metric is provided by device
+		Necessity: each metric is provided by at most one device.
+		Necessity: each device provides at least one metric.
+
+	Fact type: metric has cpu id
+		Necessity: each metric has at most one cpu id
+
+	Fact type: metric has cpu temp
+		Necessity: each metric has at most one cpu temp
+
+	Fact type: metric has cpu usage
+		Necessity: each metric has at most one cpu usage
+
+	Fact type: metric has memory usage
+		Necessity: each metric has at most one memory usage
+
+	Fact type: metric has memory total
+		Necessity: each metric has at most one memory total
+
+	Fact type: metric has storage block device
+		Necessity: each metric has at most one storage block device
+
+	Fact type: metric has storage usage
+		Necessity: each metric has at most one storage usage
+
+	Fact type: metric has storage total
+		Necessity: each metric has at most one storage total
+
+	Fact type: device is undervolted
+	-- Non-optional system check, so this is not part of the metric category
+	-- Alternatively, distinguish between metrics vs checks in "metric" term
+
+
+
+-- SERVICE
+
+Term: created at timestamp
+	Concept Type: Date Time (Type)
+
+Term: service instance
+Synonym: container
+
+	Fact type: service instance has status
+		Necessity: each service instance has exactly one status.
+		Definition: "Running" or "Stopping" or "Stopped"
+		-- TODO: these are partially taken from v2 API reference which may not be ideal. Decide on status strings for v3.
+
+	Fact type: service instance has name
+		Necessity: each service instance has exactly one name.
+
+	Fact type: service instance is healthy
+	-- GET /v1/healthy
+	-- app health is a computed term comprised of all service healthcheck statuses
+
+	Fact type: service instance has created at timestamp
+		Necessity: each service instance has exactly one created at timestamp.
+
+
+-- COMMIT
+
+Term: commit
+	Concept Type: Short Text (Type)
+
+	Fact type: commit has uuid
```

Comment:
Will this create a commit table? Should commit just be a synonym for uuid (or something like that?)